core/src/avm2/globals/flash/display/LoaderInfo.as

```diff
@@ -27,5 +32,36 @@ package flash.display {
         public native function get uncaughtErrorEvents():UncaughtErrorEvents;
         public native function get sameDomain():Boolean;
         public native function get childAllowsParent():Boolean;
+
+        override public function dispatchEvent(event:Event):Boolean {
+            throw new IllegalOperationError("Error #2118: The LoaderInfo class does not implement this method.", 2118);
+        }
+
+        public static function getLoaderInfoByDefinition(object:Object):LoaderInfo {
+            // Docs say that this returns `null` when debugging isn't enabled,
+            // and (TODO) in FP it throws a SecurityError unless called in a
+            // local SWF that has been marked as "trusted", so do we really need
+            // to implement it?
+            return null;
```

Comment:
I'd include a stub warning, when somehow this breaks some SWF, it will be easier to debug.